dask/dataframe/io/tests/test_sql.py

```diff
@@ -59,6 +59,43 @@ def test_empty(db):
         assert pd_dataframe.empty is True
 
 
+@pytest.mark.filterwarnings(
+    "ignore:The default dtype for empty Series " "will be 'object' instead of 'float64'"
+)
```

Comment:
I don't like adding this here, however I don't have a better idea on how to fix it.
The warning comes from here:

https://github.com/dask/dask/blob/29e17a05f605a5ad5b8e308540387f489434cde6/dask/dataframe/io/sql.py#L151

It is a completly pandas internal issue and can be reproduced by:
```python
import pandas as pd
s = pd.Series()
s.memory_usage()
```
Unfortunately there is no arg / kwarg to `memory_usage` to accept the future behavior.

Reply:
Feel free to filter right in the code, since users wouldn't want to see this either.